specification/source/cdl.rst

```diff
@@ -1688,118 +1688,75 @@ Therefore, tools that process CDL can infer the following information:
 
 .. _sec_tag_pro:
 
-Tagged Properties
+Semantic Information
 .................
 
-The buildings industry uses different tagging schemes such as
-Brick (https://brickschema.org/) and Haystack (https://project-haystack.org/).
-CDL allows, but does not require, use of the Brick or Haystack tagging scheme.
+The buildings industry uses different metadata schemes such as
+`Brick <https://brickschema.org/>`_, `Project Haystack <https://project-haystack.org/>`_ and `ASHRAE 223p <https://www.ashrae.org/about/news/2018/ashrae-s-bacnet-committee-project-haystack-and-brick-schema-collaborating-to-provide-unified-data-semantic-modeling-solution>`_. CDL allows, but does not require, use of the Brick or Haystack or any other tagging scheme.
 
-CDL allows to add tags to declarations that instantiate
+* Semantic information will be included within the CDL ``annotation`` keyword, using the ``__semantic`` annotation. Every element in CDL can have an annotation and correspondingly, semantic information
+ * Inputs
+ * Outputs
+ * Parameters
+ * CDL blocks: both elementary blocks (:numref:`sec_ele_blo`), and composite blocks (:numref:`sec_com_blo`)
```

Comment:
Elementary blocks are in an immutable library, which should be write protected. Hence we cannot add an annotation. We can however instantiate this in a composite block and add there the annotation. That should suffice in my view. Therefore, I think the "elementary block" needs to be removed from this list.